README.rst

```diff
@@ -49,7 +49,7 @@ The high-level overview of the architecture is:
     technically (each driver can be created as a separate App talking to
     the Connector App through some defined interface). A list of readers
     supported by CCID can be found on their website
-    <https://pcsclite.alioth.debian.org/ccid/section.html>.
+    <https://ccid.apdu.fr/#readers>.
```

Comment:
The exact equivalent of the old link is "https://ccid.apdu.fr/ccid/section.html", let's use this.

Reply:
Ping - I don't see a reason to switch from a direct link to the devices information.